### ISSUE TITLE
[ci] [appveyor] propose using 'rolling-builds' to reduce queue size

### ISSUE THREAD

**Issue by vince-fugnitto** [theia-ide/theia#3581]
**AppVeyor** is often queued and what makes matters worse is that if a pull request has been amended, AppVeyor queues up each amend instead of building the latest changes.

I propose we instead use [Rolling Builds](https://www.appveyor.com/docs/build-configuration/#rolling-builds) option. `Rolling Builds` ensures that old commits for the same branch are cancelled if there is a newer commit thus reducing the queue size. 

In other words, **rolling builds make sure that only the most recent commit is built.**

**Comment by vince-fugnitto** [comment]
@kittaakos what do you think?

**Comment by marcdumais-work** [comment]
I do not see a downside to enabling this option.

**Comment by vince-fugnitto** [comment]
> I do not see a downside to enabling this option.

 It seems as though we are currently making use of such a feature in `Travis` with the use of auto-cancellation.

<img width="855" alt="screen shot 2018-11-21 at 5 35 28 pm" src="https://user-images.githubusercontent.com/40359487/48871742-f7601c00-edb3-11e8-812a-9512cba6b22a.png">

**Comment by kittaakos** [comment]
I have enabled the followings for the `theia` repository:
![screen shot 2018-11-22 at 10 43 38](https://user-images.githubusercontent.com/1405703/48894657-908a4380-ee43-11e8-9596-835be0716f04.jpg)

**Comment by vince-fugnitto** [comment]
thanks @kittaakos !
